<?xml version="1.0" encoding="UTF-8"?>

<launch>


    <node name="serial_node" pkg="jeju"  type="serial_node"
    />

    <!-- <node name="speed_panner" pkg="macaron_4"  type="speed_planner.py"
    output="screen"/> -->

    
</launch>